snakemake/workflow.py

```diff
@@ -739,7 +741,9 @@ def files(items):
                 if cluster or cluster_sync or drmaa:
                     logger.resources_info("Provided cluster nodes: {}".format(nodes))
                 else:
-                    warning = "" if cores > 1 else " (use --cores to define parallelism)"
+                    warning = (
+                        "" if cores > 1 else " (use --cores to define parallelism)"
+                    )
```

Comment:
Has black done this? Mine was fine with the original formatting, strange...

Reply:
I believe so, it didn't pass CI here the first time around, and I had to run black on this particular script.

Reply:
How have you found using black? It works well? I read about it when it came out, and it seemed like a great idea (akin to go fmt) but I was nervous to let it loose on code.

Reply:
It works fine. I recently had it reformat the entire Snakmeake codebase without issues. Very neat.